interfaces/builtin/greengrass_support.go

```diff
@@ -160,21 +291,38 @@ umount /var/snap/@{SNAP_NAME}/**,
 # sandbox for now.
 /lambda/ r,
 /lambda/** ixr,
+
+# the python runtime tries to access /etc/debian_version, presumably to identify what system it's running on
+# note there may be other accesses that the containers try to run...
+/etc/ r,
+/etc/debian_version r,
+
+# the java runtime attempts to access this
+/etc/ssl/certs/java/{,*} r,
```

Comment:
It is weird this isn't in the abstraction... :\

Reply:
Indeed it is odd. Perhaps a bug to file with apparmor?

Reply:
Yeah, https://bugs.launchpad.net/apparmor/+bug/1816372

Reply:
I added a comment about the bug to the access here.